SPECS/pcsc-lite/pcsc-lite.signatures.json

```diff
@@ -2,7 +2,7 @@
  "Signatures": {
   "gpgkey-F5E11B9FFE911146F41D953D78A1B4DFE8F9C57E.gpg": "204e5824015b70baa1da46a58aa3b2be53e67e03848e180006c82edca387c6da",
   "org.debian.pcsc-lite.policy": "4e39100040bd847d663748d73babe6c6c777ac2e579773ee73b9738068729c67",
-  "pcsc-lite-1.9.0.tar.bz2": "0148d403137124552c5d0f10f8cdab2cbb8dfc7c6ce75e018faf667be34f2ef9",
-  "pcsc-lite-1.9.0.tar.bz2.asc": "298fda0205d65d19d2c28c3af2f390d9b62429e53cb28bfb064321a83c69ef3e"
+  "pcsc-lite-1.9.5.tar.bz2": "9ee3f9b333537562177893559ad4f7b8d5c23ebe828eef53056c02db14049d08",
+  "pcsc-lite-1.9.5.tar.bz2.asc": "c6079ed5e6caa3c9f8259da2040274acb4c35119ec6be39558a80a9463e6b096"
```

Comment:
As part of the `SPEC` move clean-up please also remove the `*.asc` and `*.gpg` sources + any code using them inside the specs. We validate the sources through the signatures and don't need to keep these files.

Reply:
Removed